README.md

```diff
@@ -30,5 +30,5 @@ Alternatively, use the `spin pluginify` plugin to install from a fresh build. Th
 ```sh
 spin plugins install pluginify
 cargo build --release
-spin pluginify install
+spin pluginify
```

Comment:
This just creates the plugin, need the install argument to actually install it.

Reply:
Ok I pushed a change.